controller/integration/env/common.go

```diff
@@ -0,0 +1,25 @@
+package env
+
+import (
+	"fmt"
+	"os"
+)
+
+const (
+	EnvVarE2EKubeconfig = "E2E_KUBECONFIG"
```

Comment:
IMO this should be `INTEGRATION_KUBECONFIG`. We agreed to not call those tests e2e anymore.

Reply:
In this context I think the shorter name is useful. The same way we use G8sClient but Giantnetes is not a thing.

I'd be OK with KIND_KUBECONFIG but then we need to change it if we move to something else later.

Reply:
`I8N_KUBECONFIG` :D Don't know.

Reply:
It's in the `env` package under the `integration` folder, maybe `E2E` or `I8N` are redundant. It's the tenant cluster kubeconfig, right? What about `TC_KUBECONFIG`?

Reply:
https://gigantic.slack.com/archives/CUTGJ4NKA/p1584529656034000